how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,783 @@
+        "Use the neptune.init() method to:\n",
+        "* Re-open a run using the ID you got from the previous step \n",
+        "* Re-open it in the `read-only` mode\n",
+        "\n",
+        "You use the `read-only` mode so the metadata previously logged to the run is not accidentally changed. Also, you can re-open a run as many times as needed.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "YIAK4NasfQ_f",
+        "outputId": "0ba95ae9-6520-4323-f67a-a9d6bdcb42da"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "https://app.neptune.ai/common/showroom/e/SHOW-3295\n",
+            "Remember to stop your run once you’ve finished logging your metadata (https://docs.neptune.ai/api-reference/run#stop). It will be stopped automatically only when the notebook kernel/interactive console is terminated.\n"
+          ]
+        }
+      ],
+      "source": [
+        "failed_run = neptune.init(\n",
+        "    project=\"common/showroom\",\n",
+        "    api_token=\"ANONYMOUS\",\n",
+        "    mode=\"read-only\",\n",
+        "    run=failed_run_id\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-WbPjLDk1GcC"
+      },
+      "source": [
+        "## Step 3: Fetching and downloading data from Neptune\n",
+        "\n",
+        "Fetch metadata(dataset and hyperparameters) needed to re-run the training. Precisely, you will download the hyperparameters used in the failed run to instantiate a model with the same configuration and then you will download the dataset path to get the same dataset too.\n",
+        "\n",
+        "To do that:"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_E_T5gUvA6kP"
+      },
+      "source": [
+        "### Use the  [.download()](https://docs.neptune.ai/api-reference/field-types#.download-3) method to retrieve the [dataset artifact](https://docs.neptune.ai/api-reference/field-types#artifact) to your local disk:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XcfHQqomSrAT"
+      },
+      "outputs": [],
+      "source": [
+        "data_dir = 'data'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "metadata": {
+        "id": "8j0cP5ZTA5AJ"
+      },
+      "outputs": [],
+      "source": [
+        "failed_run['artifacts/dataset'].download(destination=data_dir)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BVJKNQxIYPAC"
+      },
+      "source": [
+        "### Use the [.fetch()](https://docs.neptune.ai/api-reference/field-types#.fetch-1) method to retrieve hyperparameters:"
```

Comment:
same here, pls drop `.` 

I tihnk overall you can reference methods without prefixing with `.`